src/i18n/en.json

```diff
@@ -822,5 +830,97 @@
   {
     "id": "Back to home page",
     "translation": "Back to home page"
+  },
+  {
+    "id": "getinvolved_description",
+    "translation": "Wondering how to get involved with Decred? We have a whole list of projects for you to contribute to. Get familiar with the project your interested in, hack on it, and send us a pull request!"
```

Comment:
```suggestion
    "translation": "Wondering how to get involved with Decred? We have a whole list of projects for you to contribute to. Get familiar with the project you're interested in, hack on it, and send us a pull request!"
```

Reply:
Can also add link from "get involved" to new docs page on contributing to Decred, once it's live.